integration/images.go

```diff
@@ -247,11 +247,17 @@ func (d *DockerFileBuilder) buildCachedImages(imageRepo, cacheRepo, dockerfilesP
 	cacheFlag := "--cache=true"
 
 	for dockerfile := range d.TestCacheDockerfiles {
+		benchmarkEnv := "BENCHMARK_FILE=false"
+		if b, err := strconv.ParseBool(os.Getenv("BENCHMARK")); err != nil && b {
```

Comment:
I'm not sure if the condition here `err != nil && b` could actually happen, from the code [here](https://golang.org/src/strconv/atob.go?s=351:391#L1), unless I'm missing something?

Reply:
oops, it should be `err == nil && b`, nice catch